AsyncDisplayKit/ASNetworkImageNode.mm

```diff
@@ -415,16 +424,16 @@ - (void)_lazilyLoadImageIfNecessary
         NSUUID *cacheUUID = [NSUUID UUID];
         _cacheUUID = cacheUUID;
 
-        void (^cacheCompletion)(UIImage *) = ^(UIImage *image) {
+        void (^cacheCompletion)(id <ASImageContainerProtocol>) = ^(id <ASImageContainerProtocol> imageContainer) {
           // If the cache UUID changed, that means this request was cancelled.
           if (![_cacheUUID isEqual:cacheUUID]) {
             return;
           }
           
-          if (image == NULL && _downloader != nil) {
+          if ([imageContainer asdk_image] == NULL && _downloader != nil) {
```

Comment:
It seems a bit silly that the original conditions were comparing against NULL in the first place, I guess this code used to use CGImageRef?  Time for nil!